<launch>
	<!-- launches controls node with appropriate parameters. Assumes Simulator with Roscore is already running. -->	
    <rosparam command="load" file="$(find controls)/controlsParameters.yaml"/>   <!-- Parameters needed for controls to work at all!!! -->
  <!-- Controls node -->
	 <node name="thrust_mapper" pkg="controls" type="thrust_mapper"  respawn="false" output="screen"/>
</launch>
